integrations/api_repo_test.go

```diff
@@ -28,3 +28,11 @@ func TestAPISearchRepoNotLogin(t *testing.T) {
 	resp := MakeRequest(req)
 	assert.EqualValues(t, http.StatusOK, resp.HeaderCode)
 }
+
+func TestAPIOrgReposNotLogin(t *testing.T) {
+	assert.NoError(t, models.LoadFixtures())
+
+	req := NewRequest(t, "GET", "/api/v1/orgs/org1/repos")
```

Comment:
@awwalker  You need to be authenticated for this API endpoint; that's why the test is failing. Use the `loginUser(..)` helper function, and take a look at other API integrations test (a lot of them already use `loginUser(..)`).